exercise3.py

```diff
@@ -0,0 +1,16 @@
+class SportsArray:
+
+
+    def skip_sports(arr,skip):
+        try:
+            if len(arr) < 1 :
+                raise AttributeError
```

Comment:
could be simplified using a single if, also you could raise it with a message

Reply:
Done